package.json

```diff
@@ -77,22 +82,24 @@
     "gh-pages": "^2.0.1",
     "glob": "^7.1.2",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^23.1.0",
+    "jest": "^24.8.0",
     "lerna": "^2.1.2",
     "link-parent-bin": "^1.0.0",
     "markdown-magic": "^0.1.25",
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
-    "react-router-dom": "^4.0.0",
+    "react-router-dom": "^5.0.0",
```

Comment:
remove?

Reply:
Let me try it. I have included it in terra-clinical as well. I think it was causing some issues, though it might be that a package is defining their own version and its getting pulled to the top.

Reply:
it renders white screen without it

Reply:
weird